.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "cnmem"]
+	path = cnmem
+	url = https://github.com/NVIDIA/cnmem.git
```

Comment:
@borisfom are we sure we want to go with a submodule? It took me a little while to figure out how to sync this code, and I like to think of myself as more git-savvy than most. I know that people in torch-land are comfortable with submodules, but this is new territory for caffe.

That being said, I can't think of any better solutions.